docs/guides/miscellaneous/how_to_configure_notebooks_generated_by_suite_edit.md

```diff
@@ -2,4 +2,109 @@
 title: How to configure notebooks generated by suite-edit
 ---
 
-This article is a stub.
+This guide shows how to customize the notebooks generated by `great_expectations suite edit`
+
+:::note This how-to guide assumes you have already:
+  - [Set up a working deployment of Great Expectations](../../tutorials/getting_started/intro.md)
+:::
```

Comment:
```suggestion
import Prerequisites from '../connecting_to_your_data/components/prerequisites.jsx'

This guide shows how to customize the notebooks generated by `great_expectations suite edit`

<Prerequisites>
  
</Prerequisites>
:::
```
Please note that we have a `<Prerequisites>` React component that we're using in Docusaurus. It automatically adds "This how-to guide assumes you have already" as well as a few general prerequisites (I believe one of the defaults is the working deployment). You'll need to import it out of `connecting_to_your_data/components`